web/src/beta/components/Resizable/hooks.ts

```diff
@@ -131,7 +176,16 @@ export default (
     }
     bindEventListeners();
     return () => unbindEventListeners();
-  }, [bindEventListeners, isResizing, unbindEventListeners]);
+  }, [
+    bindEventListeners,
+    difference,
+    isResizing,
+    localStorageKey,
+    minimized,
+    size,
+    startingSize,
+    unbindEventListeners,
```

Comment:
Same here, curious why you are adding so many deps. Any deps already attached to bindEventListeners or unbindEventListeners when changed will update those callbacks, which in turn will refire this useEffect.
If you add them here too, I am not 100% sure, but it might make this useEffect fire MANY times more than necessary.